src/app/core/data/dso-redirect-data.service.ts

```diff
@@ -56,7 +56,7 @@ export class DsoRedirectDataService extends DataService<any> {
   findByIdAndIDType(id: string, identifierType = IdentifierType.UUID): Observable<RemoteData<FindByIDRequest>> {
     this.setLinkPath(identifierType);
     return this.findById(id).pipe(
-      getFinishedRemoteData(),
```

Comment:
@YanaDePauw : Tiny code review note... LGTM rightly notes that we should remove `getFinishedRemoteData()` from the import statements in this service, as it's no longer used.

Reply:
done